python/ray/data/_internal/execution/operators/base_physical_operator.py

```diff
@@ -16,7 +16,12 @@
 class InternalQueueOperatorMixin(PhysicalOperator, abc.ABC):
     @abc.abstractmethod
     def internal_queue_size(self) -> int:
-        """Returns Operator's internal queue size"""
+        """Returns Operator's internal queue size (in blocks)"""
+        ...
```

Comment:
What are we hoping to achieve by changing the unit of `internal_queue_size` from bundles to blocks?